js/utils/string.js

```diff
@@ -0,0 +1,7 @@
+export function capitalize(str) {
+  if (typeof str !== 'string') {
+    throw new Error('Please provide a str as a String.');
+  }
+
+  return str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
At this point our string utils consists of a single capitalize function, so it's not looking like we need a full blow lib. I want to be mindful of bringing in libs since it impacts download time of the binary.

Anyhow, thanks for pointing out that lib. I'll keep it in mind and if we see that we are in need of more than a small number of functions that the lib provides, we could look into bringing it in.